### DISCUSSION TITLE
tracing-opentelemetry version in the source code

### DISCUSSION THREAD

**Discussion by abdolence** [tokio-rs/tracing#2315] [Q&A] [answered]
Hey,

https://github.com/tokio-rs/tracing/blob/a77be16e395893a82474688e2f3f54522937e64c/tracing-opentelemetry/Cargo.toml#L3

It is already `0.17.x` is available as a crate, but the source code seems misleading with `0.16`. Is this is a mistake or I'm looking at the wrong place?

**Comment by hawkw** [comment, marked as answer]
The `master` branch [is an in-progress development branch for v0.2.x of `tracing`](https://github.com/tokio-rs/tracing#the-master-branch-is-the-pre-release-development-version-of-tracing-please-see-the-v01x-branch-for-the-versions-of-tracing-released-to-cratesio).  Release versions of `tracing` crates are published from the [`v0.1.x` branch](https://github.com/tokio-rs/tracing/tree/v0.1.x). 

If you check the `v0.1.x` branch's version of `tracing-opentelemetry`, you'll see that [the latest release version is present in its `Cargo.toml`](https://github.com/tokio-rs/tracing/blob/v0.1.x/tracing-opentelemetry/Cargo.toml#L3). We haven't bothered updating the versions on the `tracing` 0.2 development branch, because releases aren't published from that branch yet.

Hope that helps!
